Description: Upstream changes introduced in version 1.0-1ubuntu2
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 t10-keyboard (1.0-1ubuntu2) natty; urgency=low
 .
   * added missing dep git (which actually wont do anything...)
 .
 The person named in the Author field signed this changelog entry.
Author: Nicolai Ommer <nicolai.ommer@gmail.com>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- t10-keyboard-1.0.orig/src/res/version
+++ t10-keyboard-1.0/src/res/version
@@ -1 +1 @@
-v1.0-4-g0238aea
+v1.0-11-g6a334ff
